packages/utils/src/tabListener.ts

```diff
@@ -0,0 +1,30 @@
+const navKeys = ["Tab", "ArrowUp", "ArrowRight", "ArrowDown", "ArrowLeft"];
```

Comment:
Do we alle the arrow-keys? I don't think you can use arrow keys to change focused item?

Reply:
They are used to navigate radio buttons, tabs, dropdowns and a few other structures: https://webaim.org/techniques/keyboard/#testing